simulationdataschema/model_system.py

```diff
@@ -761,37 +740,44 @@ def normalize(self, archive, logger):
 
 
 class ModelSystem(System):
+    # ! Work in this description with new `AtomState`
     """
-    Model system used as an input for the computation. It inherits from `System` where a set
-    of sub-sections for the `elemental_composition` is defined.
+    Model system used as an input for the computation.
 
-    We also defined:
+    We define:
         - `name` refers to all the verbose and user-dependent naming in ModelSystem,
         - `type` refers to the type of the ModelSystem (atom, bulk, surface, etc.),
-        - `dimensionality` refers to the dimensionality of the ModelSystem (0D, 1D, 2D, 3D),
+        - `dimensionality` refers to the dimensionality of the ModelSystem (0, 1, 2, 3),
 
-    If the ModelSystem `is_representative`, the normalization occurs. The time evolution of
-    the system is encoded on the fact that ModelSystem is a list under Simulation, and for
-    each element of that list, `time_step` can be defined.
+    If the ModelSystem `is_representative`, normalization occurs. The time evolution of the
+    ModelSystem is encoded on the fact that it is a list under `Simulation`, and for each element
+    of that list, `time_step` can be defined.
```

Comment:
Is `time_step` an integer step or a float physical time? Right now in calculation there is both `step` and `time`, which is also able to cover consecutive non-time-dependent workflows like geom. opts. I guess we should continue this approach and use `step` for the referencing between system and calculation (if this will still be relevant), and any other normalization that may be relevant. We will also need to store the physical time in some cases, but this will not always be available. Is this consistent with your view?

Reply:
Yeah, thanks for bringing it up, I am aware of these quantities; I will add a note to check back this again once we start with transfering the `Outputs`.

Is that OK? (an emoji " 👍🏻  " would do 😄 )